README.md

```diff
@@ -36,9 +36,11 @@ If you run into a bug when using librespot, please search the existing issues be
 # Building
 Rust 1.32.0 or later is required to build librespot.
```

Comment:
Should we just link to COMPILING.md for all of this section now?

Reply:
Probably. We should also change the `Rust x.x.x or later is required`. To something like `The minimum version of rust required can be found here.` and link through to the minimum version specified in the travis.yml file.

Reply:
Sorry, I didn't update the Readme, will do it this evening. 
> 
> 
> Probably. We should also change the `Rust x.x.x or later is required`. To something like `The minimum version of rust required can be found here.` and link through to the minimum version specified in the travis.yml file.

I would rather people bump the readme when they bump Travis, rather than the other way around. Also if/when we move to Github actions/CircleCI or any other CI, it becomes another broken link if we aren't careful.